core/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestTags.scala

```diff
@@ -9,4 +9,8 @@ object LeonardoTestTags {
   // intent: do not run in given execution context
   // this is also synonmous with 'azure test'
   object ExcludeFromJenkins extends Tag("ExcludeFromJenkins")
+
+  // intent: do not run on every PR commit
+  // e.g. GPU tests should not run at every PR commit to avoid resource allocation problems
+  object ExcludeFromPRTest extends Tag("ExcludeFromPRTest")
```

Comment:
I'd name this ExcludeFromPrCommit to clarify that it's ignoring commits to PR branches. I think the -Test suffix isn't necessary.

Reply:
Good idea, will do :)